Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM node:16.13-alpine
```

Comment:
Alpline is really good for production but for dev you may be missing some good utilities to log in. Feel free to abort out of it.